submissions/evgenii-del/memory-game/css/style.css

```diff
@@ -0,0 +1,70 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.wrapper {
+    max-width: 800px;
+    margin: 0 auto;
+}
+
+/* Cards */
+
+.cards {
+    display: flex;
+    flex-wrap: wrap;
+    margin-top: 30px;
+}
+
+/* Flipper */
+
+.flip-container {
+    max-width: calc(100% / 4 - 10px);
+    perspective: 1000px;
+    overflow: hidden;
+    margin: 5px;
+}
+
+.flip-container, .front, .back {
+    width: 100%;
+    height: 200px;
+}
+
+.flipper {
+    transition: 0.6s;
+    transform-style: preserve-3d;
+    position: relative;
+}
+
+.flipper .front, .back {
+    backface-visibility: hidden;
+    position: absolute;
+    top: 0;
+    left: 0;
+    pointer-events: none;
+}
+
+.flipper .front {
+    z-index: 2;
+    transform: rotateY(0deg);
+    position: relative;
+}
+
+.flipper .back {
+    transform: rotateY(180deg);
+}
+
+.flipper img {
+    width: 100%;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    margin-right: -50%;
+    transform: translate(-50%, -50%);
+    pointer-events: none;
+}
+
+.flipped {
+    transform: rotateY(180deg);
+}
```

Comment:
Don't forget to configure your IDE for adding an empty line in the end of each file.